stable/netsil/templates/deployment.yaml

```diff
@@ -0,0 +1,95 @@
+apiVersion: apps/v1beta1
+kind: Deployment
+metadata:
+  name: {{ template "fullname" . }}-stream-processor
+  labels:
+    app: {{ template "name" . }}
+    heritage: {{ .Release.Service }}
+    release: {{ .Release.Name }}
+    chart: {{ .Chart.Name }}-{{ .Chart.Version }}
+    component: stream-processor
+spec:
+  replicas: 1
+  template:
+    metadata:
+      name: stream-processor
+      labels:
+        app: {{ .Chart.Name }}
+        component: stream-processor
+        version: stable
```

Comment:
I don't think the version label makes sense, especially since it is hard-coded. I'd remove it.

Reply:
done.